source/server-selection/tests/in_window/many-choices.yml

```diff
@@ -0,0 +1,60 @@
+description: Selections from many choices occur at correct frequencies
```

Comment:
@ShaneHarvey 

This test includes some really small frequencies, so I upped the iteration count to 10,000 and decreased the threshold to 2%. Let me know if this works for your runner okay and doesn't take too long to run.

Reply:
Note that even with 10000 iterations I got this false positive [in this evergreen patch](https://evergreen.mongodb.com/version/5fadb207c9ec440ad07f3cb1):
```
 [2020/11/12 22:16:08.832] FAIL [0.230s]: test_in_window_one_least_two_tied_Least_operations_gets_most_selections,_two_tied_share_the_rest (test_server_selection_in_window.TestAllScenarios)
 [2020/11/12 22:16:08.832] ----------------------------------------------------------------------
 [2020/11/12 22:16:08.832] Traceback (most recent call last):
 [2020/11/12 22:16:08.832]   File "/data/mci/491f6078eca3c0414292a7732afbf55b/src/test/__init__.py", line 444, in wrap
 [2020/11/12 22:16:08.832]     return f(*args, **kwargs)
 [2020/11/12 22:16:08.832]   File "/data/mci/491f6078eca3c0414292a7732afbf55b/src/test/test_server_selection_in_window.py", line 92, in run_scenario
 [2020/11/12 22:16:08.832]     self.run_scenario(scenario_def)
 [2020/11/12 22:16:08.832]   File "/data/mci/491f6078eca3c0414292a7732afbf55b/src/test/test_server_selection_in_window.py", line 87, in run_scenario
 [2020/11/12 22:16:08.832]     self.assertAlmostEqual(actual_freq, freq, delta=0.02)
 [2020/11/12 22:16:08.832] AssertionError: 0.6805 != 0.66 within 0.02 delta (0.020499999999999963 difference)
```

Reply:
So we can still implement this proposal, let's just increase `outcome.tolerance` for this test and/or reduce the number of servers in the topology.

Reply:
Increased tolerance to 0.03, which seems to work fine for me. Let me know if that has any issues for your runner.

Note: I left `iterations` outside of `outcome` since it had more to do with running the test then inspecting the outcome. `tolerance` is under there though.